ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -109,7 +109,7 @@ function AlertDetails({ selectedAlert }: { selectedAlert: Alert }) {
       borderRadius={BorderRadius.SM}
     >
       <Text variant={TextVariant.bodySm}>{selectedAlert.message}</Text>
-      {selectedAlert.alertDetails && selectedAlert.alertDetails?.length > 0 ? (
+      {(selectedAlert.alertDetails?.length ?? 0) > 0 ? (
```

Comment:
Since `1` and above is truthy, but `0` and `undefined` is falsey, could we just go with `selectedAlert.alertDetails?.length`?